src/BusinessLayer/InternetReporting.cpp

```diff
@@ -202,7 +202,7 @@ QJsonObject InternetReporting::makeBatteryFaults()
     errorFlags.insert("OpenWiringFault", batteryFaultsData_.openWiringFault);
     errorFlags.insert("CurrentSensorFault", batteryFaultsData_.currentSensorFault);
     errorFlags.insert("PackVoltageSensorFault", batteryFaultsData_.packVoltageSensorFault);
-    errorFlags.insert("WeakPackFault", batteryFaultsData_.voltageRedundancyFault);
+    errorFlags.insert("WeakPackFault", batteryFaultsData_.weakPackFault);
```

Comment:
I don't recognize this fault, @bill-luu is this accurate?

Reply:
Yep! 
![image](https://user-images.githubusercontent.com/15304203/35116077-61d27f38-fc47-11e7-98e5-52b00676a8ed.png)